integration_test/third_party_apps_data/applications/couchdb/metadata.yaml

```diff
@@ -43,3 +44,34 @@
   monitored_resource: gce_instance
   labels:
     view: .*
+expected_logs:
+- log_name: couchdb
+  fields:
+  - name: jsonPayload.node
+    value_regex: couchdb
+    type: string
+    description: node instance name
+  - name: jsonPayload.path
+    value_regex: /forbidden
+    type: string
+    description: request path
+  - name: jsonPayload.remote_user
+    value_regex: undefined
+    type: string
+    description: Authenticated username for the request
+  - name: jsonPayload.status_message
+    value_regex: ok
+    type: string
+    description: status code message
+  - name: jsonPayload.pid
+    type: string
+    description: Process ID
+  - name: jsonPayload.message
+    type: string
+    description: Log message
+  - name: jsonPayload.host
+    type: string
+    description: host instance name
+  - name: httpRequest
+    type: object
+    description: Information about the HTTP request associated with this log entry, if applicable. It's availble in access logs.
```

Comment:
Typo in `It's availble in access logs`, but I'm wondering what that part is trying to say. Is it saying that the information originated from couchdb's access logs? Or that the user can refer to the access logs for more information? I feel like the part before this sentence sufficiently explains the `httpRequest` anyway so maybe this whole chunk can be removed. Up to you.